src/lib.rs

```diff
@@ -195,3 +195,14 @@ impl<'a, N: 'a> Iterator for NodesMut<'a, N> {
         self.range.next()
     }
 }
+
+/// Iterator over edge _references_, which keep track of the source and
+/// target.
+#[derive(Debug)]
+pub struct EdgeReference<'a, NodeId, EdgeId> {
+    pub source: &'a NodeId,
+    pub target: &'a NodeId,
```

Comment:
Maybe makes sense to use the same language as `add_edge`, namely `from/to`.

Reply:
Good point 👍